apps/costcontrol/js/utils/formatting.js

```diff
@@ -208,6 +207,34 @@ var Formatting = (function() {
     formatTimeHTML: formatTimeHTML,
 
     // Given the API information compute the human friendly minutes
-    computeTelephonyMinutes: computeTelephonyMinutes
+    computeTelephonyMinutes: computeTelephonyMinutes,
+
+    formats: {}
```

Comment:
Oh, I see you cached the formatters, not the _formats_. What I was asking was to cache the formats. Caching formatters mean you need to listen for those [events in lines 239 and 240](#diff-f80fe1f3ae9362ef69f590f6898c42d2R239) although it mean to save some overhead when displaying a date. I'm ok with caching formatters but then change the map name from `formats` to `formatters`.